app/Http/Controllers/SinkApiController.php

```diff
@@ -77,6 +78,17 @@ public function scanLocalFiles(Sink $sink): Response
         ]);
     }
 
+    /**
+     * asd
+     */
+    public function getDoc(Sink $sink): Response
+    {
+        $this->authorize('view', $sink);
+        $res = Ragnarok::getSinkHandler($sink->id)->getSinkDocumentation();
+        $scode = $res === null ? 204 : 200;
+        return response($res, $scode)->header('Content-Type', 'text/html');
```

Comment:
`$res` is often used as request result stuff, but what you get here is documentation, so e.g.
```php
        $doc = Ragnarok::getSinkHandler($sink->id)->getSinkDocumentation();
        return response($doc, $doc ? 204 : 200)->header('Content-Type', 'text/html');
```

Reply:
Fixed